explore/src/main/scala/explore/targeteditor/BrightnessesEditor.scala

```diff
@@ -217,22 +200,19 @@ sealed abstract class BrightnessesEditorBuilder[T, Props <: BrightnessesEditor[T
             }
           )
 
-        // Put it inside a form to get the SUI styles right
-        Form(as = <.div, size = Small)(
```

Comment:
I like this. putting non Form stuff in a form is a kludge. we can get styles correct with plain css

Reply:
Actually, this was removed because the code here is expected to be in a Form already (the one in SiderealTargetEditor).

Which, I think, may not have been the correct decision, it may be breaking the grid layout in large screens.

Reply:
I'll check this and address in a future PR.